loopbackio/loopback.io
# Add Bank of America Logo

Status: **OPEN FOR REVIEW**
Owner: @rosshj
Reviewers: @altsang @crandmck 
## Changes
- [x] Remove Sapient logo
- [x] Add Bank of America logo
## Jira Tickets:
- n/a
## Todos:
- [ ] Designer / Engineer +1
### Feedback:

_none so far_
## How to Test
- Take a look at the home page 'Check out who's using LoopBack!' section and make sure the Sapient logo has been replaced by the Bank of America logo.
